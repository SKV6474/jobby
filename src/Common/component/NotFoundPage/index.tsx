import React from "react";

import { NOT_FOUND_IMG } from "../../constants/Images";

import {
  NotFoundContainer,
  NotFoundContent,
  NotFoundContentDescription,
  NotFoundContentTitle,
  NotFoundImg,
  NotFoundPageWrapper,
} from "./styledComponet";

const NotFoundPage = () => {
  return (
    <NotFoundPageWrapper>
      <NotFoundContainer>
        <NotFoundImg src={NOT_FOUND_IMG} alt="Not Found"></NotFoundImg>

        <NotFoundContent>
          <NotFoundContentTitle>Page Not Found</NotFoundContentTitle>
          <NotFoundContentDescription>
            We are sorry, the page you requested could not be found
          </NotFoundContentDescription>
        </NotFoundContent>
      </NotFoundContainer>
    </NotFoundPageWrapper>
  );
};
export default NotFoundPage;
