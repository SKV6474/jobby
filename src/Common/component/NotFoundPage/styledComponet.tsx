import styled from "styled-components";

export const NotFoundContainer = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  height: 100%;
`;

export const NotFoundImg = styled.img`
  width: 450px;
  height: 450px;
  @media screen and (max-width: 576px) {
    width: 300px;
    height: 300px;
  }
`;

export const NotFoundContent = styled.div`
  text-align: center;
`;

export const NotFoundContentTitle = styled.div`
  font-size: 36px;
  height: 48px;
  font-weight: 500;
  color: #fff;
  margin: 40px 0 30px;
  @media screen and (max-width: 576px) {
    font-size: 28px;
    margin: 30px 0 20px;
  }
`;

export const NotFoundContentDescription = styled.div`
  font-size: 16px;
  height: 22px;
  color: #64748b;
  @media screen and (max-width: 390px) {
    width: 325px;
  }
`;

export const NotFoundPageWrapper = styled.div`
  height: 100vh;
  background-color: #000;
  display: flex;
  align-items: center;
  justify-content: center;
`;
