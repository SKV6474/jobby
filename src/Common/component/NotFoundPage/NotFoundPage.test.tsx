import renderer from "react-test-renderer";
import NotFoundPage from ".";

describe("Logo Img", () => {
  test("Logo Img SnapShot", () => {
    const tree = renderer.create(<NotFoundPage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
