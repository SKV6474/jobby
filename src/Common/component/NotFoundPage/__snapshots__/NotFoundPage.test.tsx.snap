// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Img Logo Img SnapShot 1`] = `
<div
  className="sc-jTrPJq etA-DwL"
>
  <div
    className="sc-beqWaB iXlEcV"
  >
    <img
      alt="Not Found"
      className="sc-gueYoa iYHUMi"
      src="https://assets.ccbp.in/frontend/react-js/jobby-app-not-found-img.png"
    />
    <div
      className="sc-dmqHEX eHzspH"
    >
      <div
        className="sc-hLseeU bBlhFm"
      >
        Page Not Found
      </div>
      <div
        className="sc-eDDNvR iCKCUL"
      >
        We are sorry, the page you requested could not be found
      </div>
    </div>
  </div>
</div>
`;
